src/WrappedMToken.sol

```diff
@@ -48,6 +48,7 @@ contract WrappedMToken is IWrappedMToken, Migratable, ERC20Extended {
         // Second slot
         uint128 lastIndex;
         bool hasEarnerDetails;
+        bool hasClaimRecipient;
```

Comment:
any benefit of storing `hasClaimRecipient` vs `checking _claimRecipients[msg.sender] != 0x0`

Reply:
```suggestion
       function hasClaimRecipient() external view returns (bool) {
         return _claimRecipients[account_] != address(0);
       }
```

Reply:
Checking `_claimRecipients` for every earner, even when there is none set, will cost more gas to load a cold storage slot. The `hasClaimRecipient` property of an earning account will already be hot when it is queried.